expo/gbe/models.py

```diff
@@ -36,8 +36,8 @@
 
 phone_regex = '(\d{3}[-\.]?\d{3}[-\.]?\d{4})'
 
-visible_bid_query = (Q(biddable_ptr__conference__status='upcoming') |
-                     Q(biddable_ptr__conference__status='ongoing'))
+visible_bid_query = (Q(biddable_ptr__b_conference__status='upcoming') |
+                     Q(biddable_ptr__b_conference__status='ongoing'))
```

Comment:
given that this works now with bids and not events, would there be a way to put this within the parent Bid class?